### ISSUE TITLE
Error after oh-my-zsh update

### ISSUE THREAD

**Issue by aaf-mike** [ohmyzsh/ohmyzsh#9764]
When running oh-my-zsh after the latest update, I get the error mentioned below.  When I look at agnoster.zsh-theme, it looks like a lot of the comments are missing the "# ".  So even if line 91 is commented, I would still run into issues with lines 94 and 98.

**Describe the bug**
/.oh-my-zsh/themes/agnoster.zsh-theme:91: parse error near `<<<'

**To Reproduce**
Steps to reproduce the behavior, for example:
1. Run from zsh, "source ~/.zshrc"

**Expected behavior**
See my prompt with the Agnoster theme

```
 89 # Context: user@hostname (who am I and where am I)
 90 prompt_context() {
 91 <<<<<<< Updated upstream
 92   if [[ "$USERNAME" != "$DEFAULT_USER" || -n "$SSH_CLIENT" ]]; then
 93     prompt_segment black default "%(!.%{%F{yellow}%}.)%n@%m"
 94 =======
 95   if [[ "$USER" != "$DEFAULT_USER" || -n "$SSH_CLIENT" ]]; then
 96     # prompt_segment black default "%(!.%{%F{yellow}%}.)%n@%m"
 97     prompt_segment green black "%(!.%{%F{yellow}%}.)%n"
 98 >>>>>>> Stashed changes
```

**Comment by mcornella** [comment]
You got a merge conflict because you modified the theme. I recommend redefining the `prompt_context` function in your zshrc file instead (after OMZ is sourced).

Put this in your zshrc file:
```zsh
prompt_context() {
  if [[ "$USERNAME" != "$DEFAULT_USER" || -n "$SSH_CLIENT" ]]; then
    prompt_segment green black "%(!.%{%F{yellow}%}.)%n"
  fi
}
```

And fix the agnoster.zsh-theme file so that it has the right content: https://github.com/ohmyzsh/ohmyzsh/blob/69507c9518f7c7889d8f47ec8e67bfda02405817/themes/agnoster.zsh-theme#L90-L94

**Comment by aaf-mike** [comment]
I didn't edit the theme myself (or know of any other scripts that may have).

What finally worked was:

`git reset --hard
git pull --rebase
git rebase --skip
git pull`

I will add your recommendations though.

**Comment by nohren** [comment]
https://github.com/ohmyzsh/ohmyzsh/wiki/Customization

For editing the theme.  

TLDR don't edit themes in the themes directory.  Copy paste the them you want to edit into the custom/theme directory.  These take precedence over themes.
